extensions/wikia/PhalanxII/services/PhalanxService.class.php

```diff
@@ -129,8 +131,19 @@ public function reload( $changed = [] ) {
 			: [];
 
 		$result = $this->sendToPhalanxDaemon( "reload", $params );
+		$queue_result = $this->sendToPhalanxQueue( $changed );
 		wfProfileOut( __METHOD__ );
-		return $result;
+		return $result && $queue_result;
+	}
+
+	private function sendToPhalanxQueue( $changed ) {
+		wfProfileIn( __METHOD__ );
```

Comment:
Explicit profiler calls should no longer be needed as we now use Tideways profiler.

Reply:
Fixed in e9262c9aeb9879ecce2904e4fb4a5bddd7a1bc3b